src/services/colors.js

```diff
@@ -0,0 +1,73 @@
+const themes = {
+  success: '#40e583',
```

Comment:
Add `primary` color. It's same as success in `vuestic` but not in other solutions.